proto/regen/ecocredit/v1/tx.proto

```diff
@@ -63,7 +64,7 @@ service Msg {
   // retired upon receipt.
   rpc Send(MsgSend) returns (MsgSendResponse);
 
-  // Retire retires a specified amount of tradable credits, removing the amount
```

Comment:
Added back in for one but not for the other? Was this intentional.

Reply:
oops, thought i got em all - https://github.com/regen-network/regen-ledger/pull/1556/commits/bdf4c94bc48cb6120d692e078383296b2e774d15